img/config.md
```javascript
const config = {
  //网页元数据
  metaData:{
    title: 'Leleo的个人主页🎉',
    description: '欢迎来到Leleo的奇妙世界！',
    keywords: 'Leleo,leleo,个人主页,个人网站',
    icon: '/favicon.ico'   //网页图标，支持外链
  },

  avatar: "/img/avatar.jpg", // 头像
  welcometitle: "Hi, I'm Leleo", // 标题

  // 颜色配置
  color: {
    themecolor: "#FFFFFF", // 主题颜色，推荐趋于亮白可带有轻微色调，例： #D1FFEC
    welcometitlecolor: "#FFFFFF", // 标题颜色 例： #7BFFC9
    turntablecolor1: "#FFFF00",  // 转盘渐变色一
    turntablecolor2: "#00FFFF"   // 转盘渐变色二
  },
  
  brightness: 85, // 背景亮度 --%
  blur: 5, // 毛玻璃模糊效果

  // 我的标签
  tags : ['乐观开朗','温柔体贴','随和亲切','冷静沉着','才思敏捷','风趣幽默','刚正不阿','善解人意'],

  // 默认背景壁纸
  background: {
    "pc":{   //pc端
      "type":"pic",   //"pic":静态壁纸;"video":动态壁纸
      "datainfo":{
        "title":"海洋女孩", 
        "preview":"/img/wallpaper/static/海洋女孩/image-pre.webp",
        "url":"/img/wallpaper/static/海洋女孩/image.png",     //当然，也可填写网络地址或壁纸api，如随机PC壁纸api："url":"https://t.mwm.moe/pc"
      },
    },
    "mobile":{   //移动端
      "type":"pic",
      "datainfo":{
        "title":"0001", 
        "preview":"/img/wallpaper/static-mobile/0001/image-pre.webp",
        "url":"/img/wallpaper/static-mobile/0001/image.png"  //同理，随机移动端壁纸："url":"https://t.mwm.moe/mp"
      }
    }
      
  },

  //极坐标图数据
  polarChart:{
    skills: ['Node', 'Vue.js', 'React', 'JavaScript', 'Java', 'Python', 'linux', 'Docker', 'MySQL', 'MongoDB', 'AWS'],
    skillPoints: [90, 85, 78, 88, 80, 78, 85, 65, 82, 78, 65],
  },

  //社交按钮，图标使用mdi图标（https://pictogrammers.com/library/mdi/），复制图标名称即可
  socialPlatformIcons: [
    {icon:"mdi-github",link:"https://www.github.com/leleo886"},
    {icon:"mdi-email",link:"mailto:leleo886@foxmail.com"},
    {icon:"mdi-qqchat",link:"https://im.qq.com/"},
    {icon:"mdi-wechat",link:"https://wx.qq.com/"},
    {icon:"mdi-youtube",link:"https://www.youtube.com"},
    {icon:"mdi-facebook",link:"https://www.facebook.com"}
  ],

  //打字机
  typeWriterStrings: [
    "如果你看到了这行字，说明我已经成功吸引到了你的注意力。",
    "顶峰的少年，给了你所有细节，你却说我不是迪迦，给不了你想要的光。",
    "心简单，世界就简单，幸福才会生长；心自由，生活就自由，到哪都有快乐。",
    "生命太短，没有时间留给遗憾，若不是终点，请微笑一直向前。"
  ],

  //音乐播放配置，采用MetingJS Api(https://github.com/metowolf/MetingJS)
  musicPlayer:{
    server: 'netease',  //服务提供商 --网易云音乐
    type: 'playlist',   //歌单类型
    id: '2028178887'  //歌单id ---> music.163.com/#/playlist?id=2028178887
  },

  //壁纸数据 -----可以将壁纸文件上传到图床获取网络直链。若想调用api，请前往脚本自行修改逻辑
  wallpaper:{
    pic:[
      {"title":"海洋女孩","preview":"/img/wallpaper/static/海洋女孩/image-pre.webp","url":"/img/wallpaper/static/海洋女孩/image.png"},
      {"title":"书房夜晚","preview":"/img/wallpaper/static/书房夜晚/image-pre.webp","url":"/img/wallpaper/static/书房夜晚/image.png"},
      {"title":"安逸舒适","preview":"/img/wallpaper/static/安逸舒适/image-pre.webp","url":"/img/wallpaper/static/安逸舒适/image.png"},
    ],
    picMobile:[
      {"title":"0001","preview":"/img/wallpaper/static-mobile/0001/image-pre.webp","url":"/img/wallpaper/static-mobile/0001/image.png"},
      {"title":"0002","preview":"/img/wallpaper/static-mobile/0002/image-pre.webp","url":"/img/wallpaper/static-mobile/0002/image.png"},
      {"title":"0003","preview":"/img/wallpaper/static-mobile/0003/image-pre.webp","url":"/img/wallpaper/static-mobile/0003/image.png"},
      {"title":"0004","preview":"/img/wallpaper/static-mobile/0004/image-pre.webp","url":"/img/wallpaper/static-mobile/0004/image.png"},
    ],
    video:[
      {
        "title":"尼尔：机械纪元 团队",
        "preview":"/img/wallpaper/dynamic/尼尔：机械纪元 团队/Nier-Automata-Team-pre.webm",
        "url":"/img/wallpaper/dynamic/尼尔：机械纪元 团队/Nier-Automata-Team.webm"
      },
      {
        "title":"向往航天的女孩",
        "preview":"/img/wallpaper/dynamic/向往航天的女孩/Toy-Aeroplane-pre.webm",
        "url":"/img/wallpaper/dynamic/向往航天的女孩/Toy-Aeroplane.webm"
      },
    ],
    videoMobile:[
      {
        "title":"幻觉镇-gaako_illust",
        "preview":"/img/wallpaper/dynamic-mobile/幻觉镇-gaako_illust/Hallucination_town-pre.mp4",
        "url":"/img/wallpaper/dynamic-mobile/幻觉镇-gaako_illust/Hallucination_town.mp4"
      },
      {
        "title":"chuva",
        "preview":"/img/wallpaper/dynamic-mobile/chuva/chuva-pre.mp4",
        "url":"/img/wallpaper/dynamic-mobile/chuva/chuva.mp4"
      },
      {
        "title":"Doodle-小猫女仆降临",
        "preview":"/img/wallpaper/dynamic-mobile/Doodle-小猫女仆降临/d12-pre.mp4",
        "url":"/img/wallpaper/dynamic-mobile/Doodle-小猫女仆降临/d12.mp4"
      },
    ],
  },

  //项目卡片 其中 字段"show"控制初始卡片的text是否展开
  projectcards:[ 
    {go:"🚀 前往",img:"/img/xmkp/preview.gif",title:"Project 1",subtitle:"1,000 miles of wonder", text:"If you see this line, I've managed to get your attention.",url:"https://leleo.top",show:false},
    {go:"🗂️ 前往",img:"/img/xmkp/preview.gif",title:"Project 2",subtitle:"2,000 miles of wonder", text:"If you see this line, I've managed to get your attention.",url:"https://leleo.top",show:false},
    {go:"📝 前往",img:"/img/xmkp/preview.gif",title:"Project 3",subtitle:"3,000 miles of wonder", text:"If you see this line, I've managed to get your attention.",url:"https://leleo.top",show:false},
    {go:"👍 前往",img:"/img/xmkp/preview.gif",title:"Project 4",subtitle:"4,000 miles of wonder", text:"If you see this line, I've managed to get your attention.",url:"https://leleo.top",show:false},
    {go:"🗃 前往",img:"/img/xmkp/preview.gif",title:"Project 5",subtitle:"5,000 miles of wonder", text:"If you see this line, I've managed to get your attention.",url:"https://leleo.top",show:false},
    {go:"🎨 前往",img:"/img/xmkp/preview.gif",title:"Project 6",subtitle:"6,000 miles of wonder", text:"If you see this line, I've managed to get your attention.",url:"https://leleo.top",show:false},
    {go:"💍 前往",img:"/img/xmkp/preview.gif",title:"Project 7",subtitle:"7,000 miles of wonder", text:"If you see this line, I've managed to get your attention.",url:"https://leleo.top",show:false},
    {go:"🔍 前往",img:"/img/xmkp/preview.gif",title:"Project 8",subtitle:"8,000 miles of wonder", text:"If you see this line, I've managed to get your attention.",url:"https://leleo.top",show:false},
  ],
  
  statement: ["备案号：XXICP备123456789号","Copyright © 2025 Leleo"],
}

export default config
```


src/config.js
const config = {
	"metaData": {
		"title": "dc的个人主页🎉",
		"description": "欢迎来到dc的奇妙世界！",
		"keywords": "dc,个人主页,个人网站",
		"icon": "/img/icon.png"
	},
	"avatar": "/img/avatar-new.jpg",
	"welcometitle": "Hi, I'm dc",
	"color": {
		"themecolor": "#FFFFFF",
		"welcometitlecolor": "#FFFFFF",
		"turntablecolor1": "#FFFF00",
		"turntablecolor2": "#00FFFF"
	},
	"brightness": 85,
	"blur": 5,
	"adminPassword": "6634",
	"tags": [
		"乐观开朗",
		"温柔体贴",
		"随和亲切",
		"冷静沉着",
		"才思敏捷",
		"风趣幽默",
		"刚正不阿",
		"善解人意"
	],
	"background": {
		"pc": {
			"type": "pic",
			"datainfo": {
				"title": "诺叶睦",
				"preview": "/img/wallpaper/dynamic/诺叶睦/preview - pre.jpg",
				"url": "/img/wallpaper/dynamic/诺叶睦/preview.jpg"	
			}
		},
		"mobile": {
			"type": "pic",
			"datainfo": {
				"title": "0001",
				"preview": "/img/wallpaper/static-mobile/0001/image-pre.webp",
				"url": "/img/wallpaper/static-mobile/0001/image.png"
			}
		}
	},
	"polarChart": {
		"skills": [
			"Vue.js",
			"React",
			"JavaScript",
			"Node",
			"Java",
			"Python",
			"linux",
			"Docker",
			"MySQL",
			"MongoDB",
			"AWS"
		],
		"skillPoints": [
			85,
			78,
			88,
			90,
			80,
			78,
			85,
			65,
			82,
			78,
			70
		]
	},
	"socialPlatformIcons": [
		{
			"icon": "mdi-github",
			"link": "https://www.github.com/dc15706507-crypto"
		},
		{
			"icon": "mdi-email",
			"link": "mailto:dc15706507@gmail.com"
		},
		{
			"icon": "mdi-qqchat",
			"link": "https://im.qq.com/"
		},
		{
			"icon": "mdi-wechat",
			"link": "https://wx.qq.com/"
		},
		{
			"icon": "mdi-youtube",
			"link": "https://www.youtube.com"
		},
		{
			"icon": "mdi-facebook",
			"link": "https://www.facebook.com"
		}
	],
	"typeWriterStrings": [
		"如果你看到了这行字，说明我已经成功吸引到了你的注意力。",
		"顶峰的少年，给了你所有细节，你却说我不是迪迦，给不了你想要的光。",
		"心简单，世界就简单，幸福才会生长；心自由，生活就自由，到哪都有快乐。",
		"生命太短，没有时间留给遗憾，若不是终点，请微笑一直向前。"
	],
	"musicPlayer": {
		"server": "netease",
		"type": "playlist",
		"id": ["2028178887", "916950336"]
	},
	"wallpaper": {
		"pic": [
			{
				"title": "海洋女孩",
				"preview": "/img/wallpaper/static/海洋女孩/image-pre.webp",
				"url": "/img/wallpaper/static/海洋女孩/image.png"
			},
			{
				"title": "书房夜晚",
				"preview": "/img/wallpaper/static/书房夜晚/image-pre.webp",
				"url": "/img/wallpaper/static/书房夜晚/image.png"
			},
			{
				"title": "安逸舒适",
				"preview": "/img/wallpaper/static/安逸舒适/image-pre.webp",
				"url": "/img/wallpaper/static/安逸舒适/image.png"
			},
			{
				"title": "jswcMaMj",
				"preview": "https://s21.ax1x.com/2025/07/23/pVGli59.md.jpg",
				"url": "https://s21.ax1x.com/2025/07/23/pVGli59.jpg"
			},
			{
				"title": "pgtTqoqq",
				"preview": "https://s21.ax1x.com/2025/07/23/pVGlmDO.md.jpg",
				"url": "https://s21.ax1x.com/2025/07/23/pVGlmDO.jpg"
			},
			{
				"title": "cvKMKhue",
				"preview": "https://s21.ax1x.com/2025/07/23/pVGlNqS.md.jpg",
				"url": "https://s21.ax1x.com/2025/07/23/pVGlNqS.jpg"
			},
			{
				"title": "XpxvQVoP",
				"preview": "https://s21.ax1x.com/2025/07/23/pVGlfIJ.md.jpg",
				"url": "https://s21.ax1x.com/2025/07/23/pVGlfIJ.jpg"
			},
			{
				"title": "fVEEjEOA",
				"preview": "https://s21.ax1x.com/2025/07/23/pVGlEgx.md.webp",
				"url": "https://s21.ax1x.com/2025/07/23/pVGlEgx.webp"
			},
			{
				"title": "jgnIKMpd",
				"preview": "https://s21.ax1x.com/2025/07/23/pVGldaQ.md.jpg",
				"url": "https://s21.ax1x.com/2025/07/23/pVGldaQ.jpg"
			},
			{
				"title": "mgqyySeh",
				"preview": "https://s21.ax1x.com/2025/07/23/pVGl82t.md.jpg",
				"url": "https://s21.ax1x.com/2025/07/23/pVGl82t.jpg"
			},
			{
				"title": "dSXZfZp",
				"preview": "https://s21.ax1x.com/2025/07/23/pVGlaVg.md.jpg",
				"url": "https://s21.ax1x.com/2025/07/23/pVGlaVg.jpg"
			}
		],
		"picMobile": [
			{
				"title": "0001",
				"preview": "/img/wallpaper/static-mobile/0001/image-pre.webp",
				"url": "/img/wallpaper/static-mobile/0001/image.png"
			},
			{
				"title": "0002",
				"preview": "/img/wallpaper/static-mobile/0002/image-pre.webp",
				"url": "/img/wallpaper/static-mobile/0002/image.png"
			},
			{
				"title": "0003",
				"preview": "/img/wallpaper/static-mobile/0003/image-pre.webp",
				"url": "/img/wallpaper/static-mobile/0003/image.png"
			},
			{
				"title": "0004",
				"preview": "/img/wallpaper/static-mobile/0004/image-pre.webp",
				"url": "/img/wallpaper/static-mobile/0004/image.png"
			},
			{
				"title": "DfNHPPcc",
				"preview": "https://s21.ax1x.com/2025/07/23/pVG1uQ0.md.jpg",
				"url": "https://s21.ax1x.com/2025/07/23/pVG1uQ0.jpg"
			},
			{
				"title": "cZZwzhis",
				"preview": "https://s21.ax1x.com/2025/07/23/pVG1Vij.md.jpg",
				"url": "https://s21.ax1x.com/2025/07/23/pVG1Vij.jpg"
			},
			{
				"title": "aANKZHPX",
				"preview": "https://s21.ax1x.com/2025/07/23/pVGlIR1.md.jpg",
				"url": "https://s21.ax1x.com/2025/07/23/pVGlIR1.jpg"
			}
		],
		"video": [
			{
				"title": "尼尔：机械纪元 团队",
				"preview": "/img/wallpaper/dynamic/尼尔：机械纪元 团队/Nier-Automata-Team-pre.webm",
				"url": "/img/wallpaper/dynamic/尼尔：机械纪元 团队/Nier-Automata-Team.webm"
			},
			{
				"title": "向往航天的女孩",
				"preview": "/img/wallpaper/dynamic/向往航天的女孩/Toy-Aeroplane-pre.webm",
				"url": "/img/wallpaper/dynamic/向往航天的女孩/Toy-Aeroplane.webm"
			},
			{
				"title": "世界很温柔《龙族》上杉绘梨衣",
				"preview": "https://www.leleo.top/img/wallpaper/dynamic/%E4%B8%96%E7%95%8C%E5%BE%88%E6%B8%A9%E6%9F%94%E3%80%8A%E9%BE%99%E6%97%8F%E3%80%8B%E4%B8%8A%E6%9D%89%E7%BB%98%E6%A2%A8%E8%A1%A3/A2EF5E85-pre.webm",
				"url": "https://www.leleo.top/img/wallpaper/dynamic/%E4%B8%96%E7%95%8C%E5%BE%88%E6%B8%A9%E6%9F%94%E3%80%8A%E9%BE%99%E6%97%8F%E3%80%8B%E4%B8%8A%E6%9D%89%E7%BB%98%E6%A2%A8%E8%A1%A3/A2EF5E85.webm"
			}
		],
		"videoMobile": [
			{
				"title": "幻觉镇-gaako_illust",
				"preview": "/img/wallpaper/dynamic-mobile/幻觉镇-gaako_illust/Hallucination_town-pre.mp4",
				"url": "/img/wallpaper/dynamic-mobile/幻觉镇-gaako_illust/Hallucination_town.mp4"
			},
			{
				"title": "chuva",
				"preview": "/img/wallpaper/dynamic-mobile/chuva/chuva-pre.mp4",
				"url": "/img/wallpaper/dynamic-mobile/chuva/chuva.mp4"
			},
			{
				"title": "Doodle-小猫女仆降临",
				"preview": "/img/wallpaper/dynamic-mobile/Doodle-小猫女仆降临/d12-pre.mp4",
				"url": "/img/wallpaper/dynamic-mobile/Doodle-小猫女仆降临/d12.mp4"
			}
		]
	},
	"projectcards": [
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/博客-favicon.jpg",
			"title": "博客",
			"subtitle": "止渊的博客",
			"text": "跳转到我的另一个网站（专门写博客的内容）",
			"url": "https://songnas.dpdns.org/",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/default-cover.jpg",
			"title": "edgetunnel 设置页面",
			"subtitle": "我已经部署好的魔法",
			"text": "还可以使用起来还算是好用，跟花钱的差不多，就是可用数量少",
			"url": "https://proxy-eky.pages.dev/admin",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/实时新闻-favicon.png",
			"title": "实时新闻",
			"subtitle": "Rss爬取新闻",
			"text": "简单爬取总结新闻的网站",
			"url": "https://newsnow.busiyi.world/",
			"show": false
		},
		{
			"go": "👍 前往",
			"img": "/img/xmkp/XN.jpg",
			"title": "XN",
			"subtitle": "XN博客站",
			"text": "有很多的内容包括所有\n免费的云服务器主机、免费的二级域名（需要积分但是每天签到都可以领取）",
			"url": "https://asianb.dpdns.org/",
			"show": false
		},
		{
			"go": "🗂️ 前往",
			"img": "/img/xmkp/default-cover.jpg",
			"title": "网盘",
			"subtitle": "OpenList网盘",
			"text": "现在使用replit部署openlist网站，进行文件储存中转，使用自搭建的服务器做内网穿透上公网；openlist来代理使其变成公网服务器网盘（openlist可以代理所有市面上的网盘）",
			"url": "https://pan.moe/home?path=%2F",
			"show": false
		},
		{
			"go": "🎨 前往",
			"img": "/img/xmkp/csdn.jpg",
			"title": "CSDN",
			"subtitle": "屎山最多的网站",
			"text": "有用但不多的东西，有的时候可以找到需要的内容，但是大部分要收费（很气人）",
			"url": "https://www.csdn.net/",
			"show": false
		},
		{
			"go": "💍 前往",
			"img": "/img/xmkp/github.jpg",
			"title": "github",
			"subtitle": "纯英文的网站，好用是真的好用",
			"text": "有用但是翻译看着真难受",
			"url": "https://github.com/",
			"show": false
		},
		{
			"go": "🔍 前往",
			"img": "/img/xmkp/vercel.jpg",
			"title": "vercel",
			"subtitle": "免费的部署静态博客的网站，使用方法简单",
			"text": "部署静态网站真舒服，但是只能从github上部署就很无语；虽然还有别的方法但是都好麻烦",
			"url": "https://vercel.com/",
			"show": false
		},
		{
			"go": "📝 前往",
			"img": "/img/xmkp/yjym.jpg",
			"title": "一级域名",
			"subtitle": "2个免费的org域名",
			"text": "可以获得免费的.org后缀的域名，但是一个GitHub绑定的账号免费版本的只能有两个域名",
			"url": "https://dash.domain.digitalplat.org/auth/login",
			"show": false
		},
		{
			"go": "🔍 前往",
			"img": "/img/xmkp/cf.jpg",
			"title": "cloudflare",
			"subtitle": "DNS代理，可以在中国使用虽然有点慢但是安全",
			"text": "DNS解析（免费），好用可以在中国使用，就是页面有些乱",
			"url": "https://dash.cloudflare.com/",
			"show": false
		},
		{
			"go": "🗃 前往",
			"img": "/img/xmkp/Bilibili.jpg",
			"title": "B站",
			"subtitle": "不会就去B站，主打一个慢慢学",
			"text": "不会就去B站搜",
			"url": "https://www.bilibili.com/",
			"show": false
		},
		{
			"go": "🔍 前往",
			"img": "/img/xmkp/mubu.jpg",
			"title": "幕布",
			"subtitle": "日常的笔记网站",
			"text": "比较好用的笔记网站，日常上课做的笔记挺多的",
			"url": "https://mubu.com/app",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/mg.jpg",
			"title": "萌国ICP备案",
			"subtitle": "ICP备案网站（第三方）",
			"text": "第三方的icp备案，里面还有网盘、社区、邮箱（全是萌国的）",
			"url": "https://icp.gov.moe/aboutus.php",
			"show": false
		},
		{
			"go": "🔍 前往",
			"img": "/img/xmkp/default-cover.jpg",
			"title": "美国身份信息",
			"subtitle": "其实啥身份都有",
			"text": "好用但是不常用，而且自带邮箱只是需要注册登入（还没试过要不要钱）",
			"url": "https://www.shenfendaquan.com/#google_vignette",
			"show": false
		},
		{
			"go": "🔍 前往",
			"img": "/img/xmkp/infinityfree-favicon.ico",
			"title": "infinityfree",
			"subtitle": "免费域名和服务器",
			"text": "不知道好不好用，但是可以申请免费的服务器和域名",
			"url": "https://dash.infinityfree.com/accounts",
			"show": false
		},
		{
			"go": "🔍 前往",
			"img": "/img/xmkp/稀土掘金-favicon.ico",
			"title": "稀土掘金",
			"subtitle": "程序员必备网站",
			"text": "跟csdn差不多但是没那么坑",
			"url": "https://juejin.cn/",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/blog-favicon.ico",
			"title": "免费梯子部署方法",
			"subtitle": "自己搭建",
			"text": "一个博主研究的自己搭建免费梯子，部署很简单，全部是使用cf来进行部署的，并且给了免费域名获取的链接\n可免费注册的CNAME域名\nDNSHE：https://www.dnshe.com/ 部分域名可托管CF，邀请码：CY483A4A4B\nDigitalPlat: https://domain.digitalplat.org/ 部分域名可托管CF\nClouDNS：https://www.cloudns.net/index/lang/chs\ndynv6：https://dynv6.com/\nZoneABC：https://zoneabc.net/",
			"url": "https://blog.cmliussss.com/p/edt2/#%F0%9F%9A%80-Edgetunnel2-0-%E5%85%A8%E6%96%B0%E7%89%88%E6%9C%AC%EF%BC%8C%E5%B0%B1%E6%98%AF%E8%BF%99%E4%B9%88%E7%AE%80%E5%8D%95%EF%BC%81-%E5%B0%B1%E6%98%AF%E8%BF%99%E4%B9%88%E6%96%B9%E4%BE%BF%EF%BC%81",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/dnshe-favicon.ico",
			"title": "DNSHE免费域名管理平台",
			"subtitle": "免费域名",
			"text": "可以获取三个免费域名并代理",
			"url": "https://my.dnshe.com/",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/itdog-favicon.ico",
			"title": "ITDOG",
			"subtitle": "在线ping",
			"text": "用于测试域名或者ip通讯与联通情况",
			"url": "https://www.itdog.cn/",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/default-cover.jpg",
			"title": "精灵学院",
			"subtitle": "购买梯子流量的网站",
			"text": "目前发现最便宜的网站",
			"url": "https://pub0.riolu.icu/",
			"show": true
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/hpg.jpg",
			"title": "黑苹果屋",
			"subtitle": "安装下载苹果系统",
			"text": "黑苹果系统全部系统，什么方法都有",
			"url": "https://imacos.top/",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/fnos-logo.png",
			"title": "我的FnOS",
			"subtitle": "我自己的服务器",
			"text": "自己的服务器",
			"url": "http://vip.xg.frp.one:38251/",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/网站线路监测-favicon.ico",
			"title": "网站线路监测",
			"subtitle": "uptimerobot网站线路的检测",
			"text": "查看是否正常启动网站",
			"url": "https://dashboard.uptimerobot.com/",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/default-cover.jpg",
			"title": "mindows",
			"subtitle": "手机平板刷windows系统的网站",
			"text": "永久免费可以查找可以刷的机型",
			"url": "https://mindows.cn/",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/图床-favicon.gif",
			"title": "图床",
			"subtitle": "免费储存照片的网站",
			"text": "最大支持单次上传5mb的图片",
			"url": "https://imgchr.com/",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/随机密码生成器-favicon.ico",
			"title": "随机密码生成器",
			"subtitle": "随机密码生成器",
			"text": "随机生成密码",
			"url": "https://www.avast.com/random-password-generator#pc",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/default-cover.jpg",
			"title": "X黑手网",
			"subtitle": "网络安全学习软件",
			"text": "里面有大量的网络安全软件还有教程",
			"url": "https://xheishou.com/",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/default-cover.jpg",
			"title": "全球新闻",
			"subtitle": "MONITOR",
			"text": "GitHub开源项目，可以提取全球的内容，随时查看国际事件",
			"url": "https://finance.worldmonitor.app/",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/chmlfrp-favicon.ico",
			"title": "ChmlFrp",
			"subtitle": "内网穿透",
			"text": "内网穿透网站自带部署免费域名和ssl认证",
			"url": "https://panel.chmlfrp.net/home",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/replit-favicon.png",
			"title": "replit服务器（alist部署）",
			"subtitle": "暂时部署的alist，一个动态虚拟免费服务器",
			"text": "不知道可以部署几个但是可以部署openlist",
			"url": "https://replit.com/",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/api-favicon.ico",
			"title": "新疆幻城网安科技公益大模型api",
			"subtitle": "新疆幻城网安科技公益大模型api",
			"text": "免费api模型",
			"url": "https://api.iamhc.cn/console",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/国家超算互联网-favicon.ico",
			"title": "国家超算互联网",
			"subtitle": "国家超算互联网",
			"text": "国家超算互联网；API购买",
			"url": "https://www.scnet.cn/ui/console/index.html#/llm/coding-plan",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/linuxmirrors-favicon.ico",
			"title": "linuxmirrors",
			"subtitle": "linuxmirrors",
			"text": "Linux系统镜像源",
			"url": "https://linuxmirrors.cn/use/",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/novmnet-favicon.ico",
			"title": "NOVM.NET",
			"subtitle": "NOVM.NET",
			"text": "免费云服务，扣除的费用是网络使用费用",
			"url": "https://novm.net/",
			"show": false
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/clawhub-favicon.ico",
			"title": "clawhub",
			"subtitle": "clawhub",
			"text": "openclaw的技能网站",
			"url": "https://clawhub.ai/",
			"show": false
		},
		{
			"go": "🦞 前往",
			"img": "/img/xmkp/lobster-ai.png",
			"title": "牛筋AI",
			"subtitle": "我的牛筋AI",
			"text": "我养的龙虾",
			"url": "http://127.0.0.1:18789/chat?session=agent%3Amain%3Aopenclaw-weixin%3Adirect%3Ao9cq803yp6ytfgjdsywx-6os0tcy%40im.wechat",
			"show": true
		},
		{
			"go": "🚀 前往",
			"img": "/img/xmkp/fnos-logo.png",
			"title": "学校飞牛服务器",
			"subtitle": "G108服务器",
			"text": "学校G108机房的飞牛NAS服务器",
			"url": "http://47.115.202.254:8866/",
			"show": true
		}
	],
	"statement": [
		"备案号：萌ICP备20260017号",
		"Copyright © 2025 dc"
	]
}

export default config
